websites/docs/src/content/docs/en/getting_started/local_development.mdx

```diff
@@ -31,20 +31,3 @@ Before you can install and use Ballerine, you will need the following:
 4. **Initialize the monorepo**: The project is set up as a monorepo. Initialize it using the following command:
 
 <PackageManagersTabs code={`run monorepo:init`}/>
-
-
-## Run one of the examples, or follow a guide from the [guide sections](https://docs.ballerine.com/en/learn/simple_kyb_guide/)
-
-**Start the KYB example**: To start the example flow, use the following command:
-
-<PackageManagersTabs code={`run kyb-manual-review-example`}/>
-
-After performing these steps, the backoffice should be running on http://localhost:5137/, the collection flow on http://localhost:5201/ and the workflow service will be accepting calls at http://localhost:3000/.
-
-The default username and password for the backoffice are:
-
-**Username:**
-<CodeBlock lang={`shell`}  code={`admin@admin.com`}/>
-
-**Password:**
-<CodeBlock lang={`shell`}  code={`admin`}/>
```

Comment:
For us having the example as part of the "installation" page makes sense considering the get started as soon as possible logic.